docs/DeveloperGuide.md

```diff
@@ -304,20 +307,41 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. ModCheck shows an error message.
 
       Use case resumes at step 2.
 
+**Use case: Filter a person by tag**
+
+**MSS**
+
+1. Student requests to filter persons by tag.
+2. ModCheck shows a list of persons with the tag.
+    
+    Use case ends.
+
+**Extensions**
+
+* 2a. There are no persons with the tag.
+
+    * 2a1. ModCheck shows an error message.
+        
+        Use case ends.
+
 *{More to be added}*
 
+
 ### Non-Functional Requirements
 
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
 2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+3.  A student with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+4.  A student can easily learn and use commands provided by MODCheck.
```

Comment:
Might be considered a DG 'bug' cos in the grading criteria: 
>Not scoped clearly (i.e., hard to decide when it has been met)

Will approve PR but might consider removing in future iterations

Reply:
Noted!